### PR TITLE
Test case earth

### PR DESCRIPTION
**Pull request by Domnyk**
Implements #22

### CODE REVIEW

**Comment by Domnyk** [comment]
Initial results:
* After just one epoch model results in AUC of 0.582589 on test set
* After 20 epochs model results in AUC of 0.205357
  * After 20 epochs there is no significant result of overfitting - loss on validation set is not 
     bigger than loss on training set

**Comment by Domnyk** [comment]
**More results:**
* After changing optimizer to Adam, I have noted an improvement. Model trained for 20 epochs 
  reached AUC of 0.5 on test set

Conclussions:
* I will change optimizer to Adam for next experiments

**Comment by Domnyk** [comment]
**More results**

* One model achieved AUC of 0.77 on test set after 7 epochs. However AUC of val set was below 0.5.
* My thesis supervisor advice is to use cross validation
* I'm unable to replicate results of this training. I will use trained model and retrain it with cross validation
